packages/webdriverio/src/commands/element/isExisting.ts

```diff
@@ -61,6 +61,6 @@ export default async function isExisting (this: WebdriverIO.Element) {
         )
     }
 
-    const command = this.isReactElement ? this.parent.react$$.bind(this.parent) : this.parent.$$.bind(this.parent)
-    return command(this.selector as string).then((res) => res.length > 0)
+    const command = this.isReactElement ? this.parent.react$.bind(this.parent) : this.parent.$.bind(this.parent)
+    return command(this.selector as string).then((res : any) => Boolean(res.elementId), () => false)
```

Comment:
We should avoid using `any`.

```suggestion
    return command(this.selector as string).then((res: WebdriverIO.Element) => Boolean(res.elementId), () => false)
```